### PR TITLE
Add Feeds UI to CI

### PR DESCRIPTION
**Pull request by rupurt**
Fixed linting for the global task in #2208

Screen Shot 2020-02-03 at 5 38 33 PM

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -216,6 +216,26 @@ jobs:
...
+      - run: yarn install
+      - save_cache:
+          name: Save Yarn Package Cache
+          key: v{{ checksum "cache.version" }}-yarn-vendor-{{ checksum "yarn.lock" }}
+          paths:
+            - /usr/local/share/.cache/yarn
+      - run:
+          name: Run Client Tests
+          command: yarn workspace @chainlink/heartbeat run test
+      - store_artifacts:
```
Comment (HenryNguyen5) [inline .circleci/config.yml]: Is this relevant for this test? I don't see it storing logs in integration

File: feeds_ui/package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "start": "npm run server",
     "start:dev": "cross-env PORT=3002 react-scripts start",
     "build": "react-scripts build",
-    "test": "jest test",
+    "test": "react-scripts test",
```
Comment (HenryNguyen5) [inline feeds_ui/package.json]: Should we keep it as jest to keep it consistent across the repo?

Comment (rupurt) [reply feeds_ui/package.json]: I switched this back in the underlying PR as it makes merging easier to keep them as close as possible. Once it's merged, deployed and we're not using the other repo then I think we should switch to this as it requires some slightly different config.

**Comment by rupurt** [comment]
@HenryNguyen5 rebased with the merged PR and resolved conflicts. Ready for review & then acceptance.

**Pull request merged**